src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -2822,6 +2822,8 @@ lseek_comm(off_t (*next_lseek)(int fd, off_t offset, int whence), int fd, off_t
 	}
 
 	file_list[fd_directed - FD_FILE_BASE]->offset = new_offset;
+	/* clear errno since it could be set elsewhere */
+	errno = 0;
```

Comment:
The man page of rewind() does not say that error should be set.

In general functions only set errno to a value if they also return -1, I've seen plenty of examples of libc functions that modify errno on success due to the way they work internally.  In this occasion I think the test is wrong.